src/main/drivers/bus_spi.c

```diff
@@ -59,6 +59,16 @@ SPIDevice spiDeviceByInstance(SPI_TypeDef *instance)
         return SPIDEV_4;
 #endif
 
+#ifdef USE_SPI_DEVICE_5
+    if (instance == SPI5)
```

Comment:
Betaflight coding style is to have blocks (`{ }`) around all conditional branches.

Reply:
it was a copy of existing code that also didn't have them, I'll fix them all.